inference_rules.adoc

```diff
@@ -193,12 +194,12 @@ The benchmark suite consists of the benchmarks shown in the following
 table. Quality and latency targets are still being finalized.
 
 |===
-|Area |Task |Model |Dataset |Quality |Latency constraint
-|Vision |Image classification |Resnet50-v1.5 |ImageNet (224x224) | ?? | ??
-|Vision |Image classification |MobileNets-v1 224 |ImageNet  (224x224) | ?? | ?? 
-|Vision |Object detection |SSD-ResNet34 |COCO (1200x1200) | ?? | ?? 
-|Vision |Object detection |SSD-MobileNets-v1 |COCO (300x300) | ?? | ?? 
-|Language |Machine translation |GMNT |WMT16 | ?? | ?? 
+|Area |Task |Model |Dataset |Quality |Server latency constraint| Multi-Stream latency constraint
+|Vision |Image classification |Resnet50-v1.5 |ImageNet (224x224) | 99% of FP32 | 15 ms | 50 ms
```

Comment:
> 99% of FP32

This should be specified in the model's native quality metric.

Reply:
I have added explicit values, taken from the inference repository READMEs, but they may not be up-to-date.